cli/src/common.rs

```diff
@@ -20,17 +24,30 @@ pub(crate) enum CliArgs {
     /// Run native node locally
     Node(native::Params),
     /// Serve compiled wasm node to be run in the browser
+    #[cfg(feature = "embedded-lumina")]
     Browser(server::Params),
 }
 
 /// Run the Lumina node.
 pub async fn run() -> Result<()> {
     let _ = dotenvy::dotenv();
-    let args = CliArgs::parse();
+
+    let args = if current_exe()?
+        .file_name()
+        .unwrap_or_default()
+        .to_string_lossy()
+        == "lumina-node"
```

Comment:
Many existing binaries do that. I wanted a way to avoid the awkward default of having to spell `lumina node` every time, despite the fact that `node` is the only choice.

Reply:
With this, the user needs to symlink `lumina` to `lumina-node` correct? In that case, we are talking about the user installs `lumina` from a package manager other than `cargo`, right?

Reply:
symlink or rename. For the symlink I think this would work with cargo too, rename probably more applicable to manual build. 
Not particularly adamant about this change, thought I'd be a small convenience thing

Reply:
ok keep it